feat(content): Culture Convos for Exotic Life: Bloodsea Blooms & Albatross Bans (#11301)
endless-sky/endless-sky: data/human/culture conversations.txt

				`	"What about 'Polecats'?"`
					goto polecats
					to display
						has "heardweasels"
						not "heardpolecats"
				`	"Albatross?"`
					goto albatross
					to display
						has "heardpolecats"
				`	(Thank him for the time and leave.)`
					goto ending
					to display
						or
							has "heardimplants"
							has "heardbadgers"
							has "heardweasels"
				`	(Turn around and leave.)`
					goto ending
					to display
						not "heardimplants"
						not "heardbadgers"
						not "heardweasels"
			label implants
			action
				set "heardimplants"
			`	The old man grunts, leaning forward so that his voice drops to a rough whisper. "Cybernetic impants ain't allowed 'round here - always cheat at cards. Can't trust a rigged hand when fate's supposed to be fair!"`
				goto nextquestion
			label badgers
			action
				set "heardbadgers"
			`	He pauses, then mutters simply, "Turnips."`
				goto nextquestion
			label turnips
			action
				set "heardturnips"
			`	He explodes. "Badgers! They'll dig up every last turnip I grow, ruining my patch faster than a cyborg cleans house. Nothing but trouble!"`
				goto nextquestion
			label madlib
			action
				set "heardmadlib"

> 			`	His lips curl into a bitter snarl as he spills a flood of exasperation, "They cause a ${pirate adjectives that work as titles}, ${pirate adjectives that work as titles}, ${pirate adjectives that work as titles} ${bad outcomes}!"`

Author: Good catch, and just changed! Ditto on the repetition being amusing.

			`	It's unclear how he expects you to respond.`
				goto nextquestion
			label weasels
			action
				set "heardweasels"
			`	He snorts dismissively. "Weasels, now, they're just tiny polecats scampering about like mischief incarnate - ain't worth their salt, and worse, troublemakers by nature!"`
				goto nextquestion
			label polecats
			action
				set "heardpolecats"
			`	His face darkens, and his voice grows weighty and ominous. "Polecats, they attract... the Albatross." For a suspended moment, the chatter ceases; the clamor of glasses and low murmurs fall silent as every patron registers the weight of that word. Then, as if on cue, the noise resumes, and life goes on as usual.`
				goto nextquestion
			label albatross

> 			`	Emboldened by the hushed reaction, you press on, "Albatross?" That single word slices through the renewed din, and in a heartbeat, the atmosphere shatters. Several patrons rise from their stools, their expressions hardening into threatening scowls. The old man, eyes distant and vacant, seems lost in thought while becomes clear that you have crossed a forbidden line.`

Reviewer: ```suggestion
			`	Emboldened by the hushed reaction, you press on, "Albatross?" That single word slices through the renewed din, and in a heartbeat, the atmosphere shatters. Several patrons rise from their stools, their expressions hardening into threatening scowls. The old man, eyes distant and vacant, seems lost in thought while it becomes clear that you have crossed a forbidden line.`
```

				goto restofhello
			label formerpirates
			`You step into one of the raucous open-air marketplaces of Bloodsea, where wary-looking vendors and armed guards barter over allegedly legal cargo amid piles of weathered crates. As you weave through the maze of noise and color, you catch sight of a man huddled behind one of the crates - he's coughing violently, blood staining his lips, the crate, and the ground beside him. Before you can react, a lean, scarred woman fixes her steely gaze on you, her face hidden beneath a pair of stained bandanas - one concealing her mouth and nose, the other wrapped around her head.`
			label restofhello

> 			`	"That man's got the Bloodsea Bloom," she says flatly, as if mentioning the weather.`

Author: The phrase "Before you can react, a lean, scarred woman fixes her steely gaze on you" is meant to convey that she notices you staring. I didn't mean to imply that there was any physical approach after you saw the man, since you're both at a comfortable conversation distance - it's more that you didn't notice her before as the blood coughing man was quite distracting. 

If you think the first line was too subtle I could add an additional phrase hammering that to the player like:
"Before you can react, a lean, scarred woman fixes her steely gaze on you, her face hidden beneath a pair of stained bandanas - one concealing her mouth and nose, the other wrapped around her head **- but there is no mistaking that you have caught her undivided attention**.`

> 			`	"That man's got the Bloodsea Bloom," she says flatly, as if mentioning the weather.`

Reviewer 2: Ah, right. It's probably fine as-is.

			choice
				`	"What's that?"`
				`	(Get back to your ship.)`
					goto declined

> 			`	She scoffs and leans in, her voice dropping to a rough whisper. "The red tide. Every summer, the bloody sea blooms, fills the air with nasty, toxic bits that chew up your lungs if you catch a whiff. No local goes near the beach when the bloom's a blooming."`

Author: [Abloom ](https://www.merriam-webster.com/dictionary/abloom)is a word, so I'll swap over to the first one. Don't think that strictly proper English is of utmost importance anyway here since it's a spoken colloquialism.

			`	Then, she suddenly screams, "Only offworlders and blind fools with a death wish swim in that poison!"`

> 			`	A wry grin floods her face, and she adds in a conversational tone, almost as an afterthought, "Now, I'll flay you alive and turn you into chum if you don't buy some fish bait. Eight racks." She slams a scarred hand into a rancid barrel beside her.`

Author: I just picked a [US-based slang term for money](https://en.wikipedia.org/wiki/Slang_terms_for_money#United_States) and thought rack was more fitting than something like grand. The in-game FL might know racks from cultural osmosis of movies/other media, if they've been to a city and heard it (particularly around casinos), and from the context here the rough scale of credits should be obvious (i.e. it doesn't mean 8 credits). For the out-of-game player, it doesn't matter any more than other vocab since you're immediately told it's 8,000 credits.

That does spur an amusing thought - I could give the player multiple options like:

1. (Give her 800 credits)
2. (Give her 8,000 credits)
3. (Give her 80,000 credits)

And have her get angry if the player gives too little, and shrug and content if they give too much. Would probably be better to change the "racks" term to a made-up one though for everyone to have the guessing-game experience (and could even randomize the results so what the word mean changes every time). What do think around that idea?

			label nextquestion
			choice
				`	"Why no 'Cybernetic Implants'?"`
					goto implants
					to display
						not "heardimplants"
				`	"What's the deal with 'Badgers'?"`
					goto badgers
					to display
						not "heardbadgers"
				`	"Turnips?"`
					to display
						has "heardbadgers"
						not "heardturnips"
					goto turnips
				`	"The sign says 'No ${plural nouns}'?"`
					goto madlib
					to display
						or
							has "heardimplants"
							has "heardturnips"
							has "heardweasels"
						not "heardmadlib"
				`	"What's wrong with 'Weasels'?"`
					goto weasels
					to display
						not "heardweasels"
				`	"What about 'Polecats'?"`
					goto polecats
					to display
						has "heardweasels"
						not "heardpolecats"
				`	"Albatross?"`
					goto albatross
					to display
						has "heardpolecats"

> 				`	(Thank him for the time and leave.)`

Reviewer 2: ```suggestion
				`	(Thank him for the answers and leave.)`
```

					goto ending
					to display
						or
							has "heardimplants"
							has "heardbadgers"
							has "heardweasels"
				`	(Turn around and leave.)`
					goto ending
					to display
						not "heardimplants"
						not "heardbadgers"
						not "heardweasels"
			label implants
			action
				set "heardimplants"

> 			`	The old man grunts, leaning forward so that his voice drops to a rough whisper. "Cybernetic impants ain't allowed 'round here - always cheat at cards. Can't trust a rigged hand when fate's supposed to be fair!"`

Reviewer 2: ```suggestion
			`	The old man grunts, leaning forward so that his voice drops to a rough whisper. "Cybernetic implants ain't allowed 'round here - always cheat at cards. Can't trust a rigged hand when fate's supposed to be fair!"`
```
I'm surprised the spellcheck didn't catch this.

Author: High-tech pants are banned

				goto nextquestion
			label badgers
			action
				set "heardbadgers"
			`	He pauses, then mutters simply, "Turnips."`
				goto nextquestion
			label turnips
			action
				set "heardturnips"
			`	He explodes. "Badgers! They'll dig up every last turnip I grow, ruining my patch faster than a cyborg cleans house. Nothing but trouble!"`
				goto nextquestion
			label madlib
			action
				set "heardmadlib"
			`	His lips curl into a bitter snarl as he spills a flood of exasperation, "They cause a ${pirate adjectives that work as titles}, ${pirate adjectives that work as titles}, ${pirate adjectives that work as titles} ${bad outcomes}!"`
			`	It's unclear how he expects you to respond.`
				goto nextquestion
			label weasels
			action
				set "heardweasels"
			`	He snorts dismissively. "Weasels, now, they're just tiny polecats scampering about like mischief incarnate - ain't worth their salt, and worse, troublemakers by nature!"`
				goto nextquestion
			label polecats
			action
				set "heardpolecats"

> 			`	His face darkens, and his voice grows weighty and ominous. "Polecats, they attract... the Albatross." For a suspended moment, the chatter ceases; the clamor of glasses and low murmurs fall silent as every patron registers the weight of that word. Then, as if on cue, the noise resumes, and life goes on as usual.`

Reviewer 2: ```suggestion
			`	His face darkens, and his voice grows weighty and ominous. "Polecats, they attract... the Albatross." For a suspended moment, the chatter ceases; the clamor of glasses and low murmurs fall silent as every patron registers the weight of that word. Then, as if on cue, the noise resumes, and the hubbub goes on as usual.`
```

				goto nextquestion
			label albatross
			`	Emboldened by the hushed reaction, you press on, "Albatross?" That single word slices through the renewed din, and in a heartbeat, the atmosphere shatters. Several patrons rise from their stools, their expressions hardening into threatening scowls. The old man, eyes distant and vacant, seems lost in thought while it becomes clear that you have crossed a forbidden line.`
			`	Rough hands grip your shoulders, and before you can protest, you're dragged toward the door. Outside, in the biting wind, the echo of their shouted threats follows you. Perhaps some stones are best left unturned.`
			label ending
			action
				clear "heardimplants"
				clear "heardbadgers"
				clear "heardturnips"
				clear "heardmadlib"
				clear "heardweasels"

> 				clear "heardpolecats"
> 
> 
> 

Reviewer 2: ```suggestion
				clear "heardpolecats"

```

			`	"Bah!" With a dismissive wave, they storm off, cursing tourists under their breath.`
				decline

mission "Exotic Life: Bloodsea"
	minor
	source Bloodsea
	to offer
		random < 10
	on offer
		conversation

> 			`You step into one of the raucous open-air marketplaces of Bloodsea, where armed vendors and hulking guards barter over stolen cargo amid piles of weathered cates. As you weave through the maze of noise and color, you catch sight of a man huddled behind one of the crates - he's coughing violently, blood staining his lips, the crate, and the ground beside him. Before you can react, a lean, scarred woman fixes her steely gaze on you, her face hidden beneath a pair of stained bandanas - one concealing her mouth and nose, the other wrapped around her head.`

Reviewer 3: ```suggestion
			`You step into one of the raucous open-air marketplaces of Bloodsea, where armed vendors and hulking guards barter over stolen cargo amid piles of weathered crates. As you weave through the maze of noise and color, you catch sight of a man huddled behind one of the crates - he's coughing violently, blood staining his lips, the crate, and the ground beside him. Before you can react, a lean, scarred woman fixes her steely gaze on you, her face hidden beneath a pair of stained bandanas - one concealing her mouth and nose, the other wrapped around her head.`
```

			`	Then, she suddenly screams, "Only offworlders and blind fools with a death wish swim in that poison!"`
			`	A wry grin floods her face, and she adds in a conversational tone, almost as an afterthought while waving a scarred hand, "Now, I'll flay you alive and turn you into chum if you don't buy some fish bait. Eight racks." She slams a rancid barrel beside her.`
			choice
				`	(Buy a barrel of rotten fish bait for 8,000 credits.)`
					to display
						credits > 8000
				`	(Run for it.)`
					goto declined
			action
						payment -8000
			`	You grit your teeth against the stench and fight back the urge to gag as you fumble over the credits to the gnarled pirate. With an irritated snort, she shoves the putrid barrel towards you, and you quickly roll the it away. Once she's out of sight, you ditch the barrel and dart back to the <ship>, leaving behind a grim and grimy reminder of this lawless port.`
				decline
			label declined

> 			`	You race through the maze of weathered crates and guarded stalls. As you push past the chaotic crowd, her bellowing laughter - a harsh, unbridled roar of contempt - echoes in your ears as you flee back to the safety of your <model>.`

Reviewer 2: You're first pushing past people, and then fleeing, but it's implied that you do that all at once. Could the sentence be clarified?

Author: How's 
```suggestion
			`	You race through the maze of weathered crates and guarded stalls. As you push past the chaotic crowd, her bellowing laughter - a harsh, unbridled roar of contempt - echoes in your ears. Eventually, you get back to the safety of your <model>.`
```

Reviewer 3: ```suggestion
			`	You race through the maze of weathered crates and guarded stalls. As you push past the chaotic crowd toward the safety of your <model>, her bellowing laughter - a harsh, unbridled roar of contempt - echoes in your ears.`
```

		"Lost in quiet rumination,"
	word
		" "
	word
		"his fingers tap the scarred wood,"
		"his eyes drift over the shadowed floor,"
		"his breath mingles with the smoky air,"
		"his silence is smothered by the conversations of bar patrons,"
	word
		" "
	word
		"awaiting your next question with a secretive look."
		"with mystery etched into every line of his face."
		"poised at the edge of revelation and reticence."
		"anticipating the spark of your next question."

mission "Exotic Life: Albatross"
	minor
	source Albatross
	to offer
		random < 10
	on offer
		conversation
			`You trudge through the biting wind to a weathered spaceport bar, its exterior marked by the signs proclaiming "No Gods, No Masters," and "No Cybernetic Implants. No Badgers, Weasels, Or Polecats." Inside, the air is heavy with shadow and smoke - worn wood panels, dim, flickering lights, and cluttered signs declaring even more peculiar prohibitions. Slouched in the corner by the heavy door, an old man sits silently, his half-lidded eyes watching the drunken banter and clinking glasses. You could approach him, and perhaps press for answers about the bar's odd proclamations.`
			label nextquestion

> 			`	${old man waiting}`

Reviewer 3: Having this line unnecessarily breaks the flow of the dialogue. It's also jarring if you end up with even a partial repetition of the phrase, arguably more so than having a single preset response.
```suggestion
```

			choice

> 				`	"Why no Cybernetic implants?"`

Author: ```suggestion
				`	"Why no 'Cybernetic Implants'?"`
```

> 			choice

Reviewer 2: Is there a reason all the questions here have capitalized nouns?

Author: The player is referring to the sign bans, which use the capitalized nouns. It also lets the ${plural nouns} not feel out of place since those are capitalized as well, which I think would be too clashy to be used otherwise.

Reviewer 3: If you are going to use the same capitalisation as the sign, then they should be treated as direct quotes and surrounded by quotation marks in the questions.

Author: Made all the quote changes so marking this as resolved.

					goto implants
					to display
						not "heardimplants"

> 				`	"What's the deal with Badgers?"`

Author: ```suggestion
				`	"What's the deal with 'Badgers'?"`
```

					goto badgers
					to display
						not "heardbadgers"
				`	"Turnips?"`
					to display
						has "heardbadgers"
						not "heardturnips"
					goto turnips

> 				`	"The sign says 'No ${plural nouns}'?"`

Author: Not sure how to do a direct quote within the direct quote of the sign. Does this work if it only directly quotes the plural noun?
```suggestion
				`	"The sign says no '${plural nouns}'?"`
```

> 				`	(Squint at one of the cluttered signs.) "It says 'No ${plural nouns}'?"`

Reviewer 2: Generally, we try to avoid having actions and dialog in the same sentence. Could this be changed to
```suggestion
				`	"The sign says 'No ${plural nouns}'?"`
```

					goto madlib
					to display
						or
							has "heardimplants"
							has "heardturnips"
							has "heardweasels"
						not "heardmadlib"

> 				`	"What's wrong with Weasels?"`

Author: ```suggestion
				`	"What's wrong with 'Weasels'?"`
```

					goto weasels
					to display
						not "heardweasels"

> 				`	"What about Polecats?"`

Author: ```suggestion
				`	"What about 'Polecats'?"`
```

					goto polecats
					to display
						has "heardweasels"
						not "heardpolecats"
				`	"Albatross?"`
					goto albatross
					to display
						has "heardpolecats"

> 				`	(Thank him for the time and leave.)`

Reviewer 3: You'll want to add a case if the player doesn't ask any questions.

Author: And done!

					goto ending
			label implants
			action
				set "heardimplants"
			`	The old man grunts, leaning forward so that his voice drops to a rough whisper. "Cybernetic impants ain't allowed 'round here - always cheat at cards. Can't trust a rigged hand when fate's supposed to be fair!"`
				goto nextquestion
			label badgers
			action
				set "heardbadgers"
			`	He pauses, then mutters simply, "Turnips."`
				goto nextquestion
			label turnips
			action
				set "heardturnips"

> 			`	He explodes into a brief tirade. "Badgers! They'll dig up every last turnip I grow, ruining my patch faster than a cyborg cleans house. Nothing but trouble!"`

Reviewer 3: Naming what he "explodes into" is redundant when you directly show the player what he says.
```suggestion
			`	He explodes. "Badgers! They'll dig up every last turnip I grow, ruining my patch faster than a cyborg cleans house. Nothing but trouble!"`
```

			`	You grit your teeth against the stench and fight back the urge to gag as you fumble over the credits to the gnarled woman. She suppresses an amused glint in her eye and, with surprising care, gently pushes the putrid barrel towards you, her calloused fingers brushing the rim in a brief, elegant flourish. You bow your head in gratitude, roll it away, and once she's out of sight, ditch the barrel and dart back to the <ship>, leaving behind a grim and grimy reminder of this unruly port.`
				decline
			label hundred
			action
				payment -800
			`	You grit your teeth against the stench and fight back the urge to gag as you fumble over the credits to the gnarled woman. Her nostrils flare, and she snatches to credits with a snarl. "Not even one rack," she spits, eyes blazing. Without warning, she hauls the barrel off the ground to slam it directly against your shin. You yelp and stagger backwards. She rakes a jagged knife through the air, roaring, "I'll carve you like a cod and string your entrails for the gulls!" You flee her and the barrel to dart back to the <ship>, her furious curses echoing behind you.`
				decline
			label thousand
			action
				payment -8000

> 			`	You grit your teeth against the stench and fight back the urge to gag as you fumble over the credits to the gnarled woman. With an irritated snort, she shoves the putrid barrel towards you, and you quickly roll it away. Once she's out of sight, you ditch the barrel and dart back to the <ship>, leaving behind a grim and grimy reminder of this unruly port.`

Author: And done!

				decline
			label declined
			`	You race through the maze of weathered crates and guarded stalls. As you push past the chaotic crowd toward the safety of your <model>, the woman's bellowing laughter - a harsh, unbridled roar of contempt - echoes in your ears.`
				decline

mission "Exotic Life: Albatross"
	minor
	source Albatross
	to offer
		random < 10
	on offer
		conversation

> 			`You trudge through the biting wind to a weathered spaceport bar, its exterior marked by signs proclaiming "No Gods, No Masters," and "No Cybernetic Implants. No Badgers, Weasels, Or Polecats." Inside, the air is heavy with shadow and smoke - worn wood panels, dim, flickering lights, and cluttered signs declaring even more peculiar prohibitions. Slouched in a corner by the heavy door, an old man sits silently, his half-lidded eyes watching the drunken banter and clinking glasses. You could approach him, and perhaps press for answers about the bar's odd proclamations.`

Author: Hmm, changing it to lowercase would then make it differ from the language at the Albatross spaceport, which capitalizes every word. I'd prefer the two to be the same since this is meant to be about that spaceport sign. Do you think it's fine to keep as upper case in that case, or better to switch both this and the spaceport to that lowercase `or`?
```
	spaceport `	The sign over the spaceport bar reads, "No Gods, No Masters." Below it in slightly smaller writing is another sign which reads, "No Cybernetic Implants. No Badgers, Weasels, Or Polecats." You have no idea what a "polecat" is.`
```

> 			`You trudge through the biting wind to a weathered spaceport bar, its exterior marked by signs proclaiming "No Gods, No Masters," and "No Cybernetic Implants. No Badgers, Weasels, Or Polecats." Inside, the air is heavy with shadow and smoke - worn wood panels, dim, flickering lights, and cluttered signs declaring even more peculiar prohibitions. Slouched in a corner by the heavy door, an old man sits silently, his half-lidded eyes watching the drunken banter and clinking glasses. You could approach him, and perhaps press for answers about the bar's odd proclamations.`

Reviewer 2: There's that, and also I think it helps convey the strangeness of the sign. This is not a normal place.

> 			`You trudge through the biting wind to a weathered spaceport bar, its exterior marked by signs proclaiming "No Gods, No Masters," and "No Cybernetic Implants. No Badgers, Weasels, Or Polecats." Inside, the air is heavy with shadow and smoke - worn wood panels, dim, flickering lights, and cluttered signs declaring even more peculiar prohibitions. Slouched in a corner by the heavy door, an old man sits silently, his half-lidded eyes watching the drunken banter and clinking glasses. You could approach him, and perhaps press for answers about the bar's odd proclamations.`

Reviewer 4: Hmm, I guess capital "Or" makes sense then.

			`	He explodes. "Badgers! They'll dig up every last turnip I grow, ruining my patch faster than a cyborg cleans house. Nothing but trouble!"`
				goto nextquestion
			label madlib
			action
				set "heardmadlib"
			`	His lips curl into a bitter snarl as he spills a flood of exasperation, "They cause a ${pirate adjectives that work as titles}, ${pirate adjectives that work as titles}, ${pirate adjectives that work as titles} ${bad outcomes}!"`
			`	It's unclear how he expects you to respond.`
				goto nextquestion
			label weasels
			action
				set "heardweasels"
			`	He snorts dismissively. "Weasels, now, they're just tiny polecats scampering about like mischief incarnate - ain't worth their salt, and worse, troublemakers by nature!"`
				goto nextquestion
			label polecats
			action
				set "heardpolecats"
			`	His face darkens, and his voice grows weighty and ominous. "Polecats, they attract... the Albatross." For a suspended moment, the chatter ceases; the clamor of glasses and low murmurs fall silent as every patron registers the weight of that word. Then, as if on cue, the noise resumes, and the hubbub goes on as usual.`
				goto nextquestion
			label albatross

> 			`	Emboldened by the hushed reaction, you press on, "Albatross?" That single word slices through the renewed din, and in a heartbeat, the atmosphere shatters. Several patrons rise from their stools, their expressions hardening into threatening scowls. The old man, eyes distant and vacant, seems lost in thought while it becomes clear that you have crossed a forbidden line.`

Reviewer 2: The player already knows what they said, so I think the first sentence is unnecessary.
```suggestion
			`	That single word slices through the renewed din, and in a heartbeat, the atmosphere shatters. Several patrons rise from their stools, their expressions hardening into threatening scowls. The old man, eyes distant and vacant, seems lost in thought while it becomes clear that you have crossed a forbidden line.`
```
